packages/frontend/component/src/components/workspace/index.css.ts

```diff
@@ -66,9 +65,6 @@ export const mainContainerStyle = style({
         },
       },
     },
-    '&[data-show-padding="true"][data-is-macos="true"]': {
```

Comment:
This shouldn't be removed, since it should be rounded for macOS when clientBorder is enabled

Reply:
This rounded corner should be implemented on the view, not here.

Reply:
But I haven’t implemented rounded corners yet, it needs to change with the border settings and os, you can implement it in your PR

Reply:
I've used `inherit` in view so that it don't need to care about whether `clientBorder` is enabled. But if you remove it here, I think it's also ok to manage this in view and I'll do that later.